modules/astrodon-build/mod.ts

```diff
@@ -64,7 +61,13 @@ export class Builder {
     Deno.mkdir(this.config.dist, { recursive: true });
 
     // Preatere the final executable
-    const final_bin_path = `${join(this.config.dist, this.info.name)}${Deno.build.os === "windows" ? ".exe" : ""}`;
+
+    const final_bin_path = `${join(this.config.dist, this.info.name)}${
```

Comment:
This looks way to magic too me. Maybe rewrite oradding a comment could help?

It seems weird too compared to the global README:

> Only Windows and Linux are supported

Reply:
It should be be more clear now. Btw, you are reading from the readme on the main branch, this PR is targeted to `feature/deno_tauri` which is a pretty big rewrite and it supports MacOS too.

Thanks for pointing out :))

Reply:
Oeps #26 ... should I close?